common/ui/ui.cpp

```diff
@@ -33,24 +33,28 @@ COPYRIGHT 1993-1999 PARALLAX SOFTWARE CORPORATION.  ALL RIGHTS RESERVED.
 #include "key.h"
 #include "ui.h"
 #include "mouse.h"
+#include "dxxerror.h"
 
 namespace dcx {
 
 static int Initialized = 0;
 
 unsigned char CBLACK,CGREY,CWHITE,CBRIGHT,CRED;
 
-grs_font_ptr ui_small_font;
+grs_font_ptr ui_small_font = nullptr;
```

Comment:
This should not be necessary.  The default constructor should initialize it to `nullptr` for you.